docs/topics/numbers.md

```diff
@@ -235,12 +279,13 @@ fun main() {
 //sampleStart
     val x = 5L / 2
     println(x == 2L)
+    // But not `x == 2`, as Long (x) cannot be compared to Int (2)
```

Comment:
Maybe this line should be added as a separate line `println(x == 2)`? So that the code comment below can be `// true` to match the style of the rest of the doc?